debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/OracleConnectorConfig.java

```diff
@@ -1387,6 +1400,13 @@ public Duration getLogMiningMaxDelay() {
         return logMiningMaxDelay;
     }
 
+    /**
+     * @return the maximum duration for a LogMiner session
+     */
+    public Duration getLogMiningMaximumSession() {
```

Comment:
Maybe `Optional<Duration>` would better express the logic in the subsequent conditions?